packages/api-client/src/team/billing/SupportedCurrency.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export enum SupportedCurrency {
+  CHF = 'CHF',
+  EUR = 'EUR',
+  USD = 'USD',
```

Comment:
We could think about using `'CHF' | 'EUR' | 'USD'` as type instead of an enum so that we don't have to import `SupportedCurrency` every time we want to use `PlanData`

Reply:
When using `getAvailablePlans` it can be filtered by currency. Thus we need it standalone :)

Reply:
You can still use the filter. The proposed type would look like this:

```ts
export type SupportedCurrency = 'CHF' | 'EUR' | 'USD';
```